bucket/nteract.json

```diff
@@ -9,7 +9,7 @@
             "hash": "sha512:9fca47dfcfea2c95bb091a3453106d936341f3bf05ed8f9731ecb54f98866ecb42d2352c476cd90ede9fb2ddd854d44f7b974ebd31f8a5fa936899f3475c2126",
             "installer": {
                 "script": [
-                    "extract_7zip \"$dir\\`$PLUGINSDIR\\app-64.7z\" \"$dir\"",
+                    "Expand-7zipArchive \"$dir\\`$PLUGINSDIR\\app-64.7z\" \"$dir\"",
```

Comment:
??
```suggestion
                    "Expand-7zipArchive \"$dir\\`$PLUGINSDIR\\app-64.7z\" \"$dir\" -Removal",
```

Reply:
The next `Remove-Item` will remove `$PLUGINSDIR`